src/server/pfs/server/pfsserver.proto

```diff
@@ -1,20 +1,40 @@
 syntax = "proto3";
 
-package pfsserver;
+package fileset;
```

Comment:
The other named seems more fitting.  It's still in the pfs server package.

Reply:
Yeah, this was a mistake when shuffling around the protos.